test/specs/plugin.filler.tests.js

```diff
@@ -223,14 +223,14 @@ describe('Plugin.filler', function() {
 			});
 
 			expect(decodedFillValues(chart)).toEqual([
-				'start', // 'origin'
-				0,       // 1 - 1
-				1,       // 1
-				1,       // 3 - 2
-				5,       // 4 + 1
-				7,       // 5 + 2
-				5,       // 6 - 1
-				'end',   // 'end'
+				'start',	// 'origin'
```

Comment:
Is there any reason to change spaces to tabs for comment alignment?

It breaks when changing editor config, for example (1 tab == 2 spaces):

![image](https://user-images.githubusercontent.com/3874900/28080094-01305d02-666b-11e7-8d36-9782c3ec6077.png)